src/main/kotlin/io/curity/identityserver/plugin/username/descriptor/UsernameAuthenticatorPluginDescriptor.kt

```diff
@@ -14,4 +15,7 @@ class UsernameAuthenticatorPluginDescriptor : AuthenticatorPluginDescriptor<User
         UsernameAuthenticatorPluginConfig::class.java
     
     override fun getPluginImplementationType(): String = "username"
+
+    override fun getRepresentationFunctions() = mapOf("authenticator/username/" +
+            UsernameAuthenticatorRequestHandler.templateName to GetRepresentationFunction::class.java)
```

Comment:
The template name used here can be relative to the authenticator's root, i.e. only `UsernameAuthenticatorRequestHandler.templateName` is needed

Reply:
Actually, that's a must.